pkg/controller.v2/controller.go

```diff
@@ -391,9 +391,18 @@ func (tc *TFJobController) reconcileTFJobs(tfjob *tfv1alpha2.TFJob) error {
 		return err
 	}
 
+	rstatus := make(map[string]v1.PodPhase)
+	psReplicas := 0
+	workerReplicas := 0
 	// Diff current active pods/services with replicas.
 	for rtype, spec := range tfjob.Spec.TFReplicaSpecs {
-		err = tc.reconcilePods(tfjob, pods, rtype, spec)
+		if rtype == tfv1alpha2.TFReplicaTypePS {
+			psReplicas = int(*spec.Replicas)
+		}
+		if rtype == tfv1alpha2.TFReplicaTypeWorker {
+			workerReplicas = int(*spec.Replicas)
+		}
```

Comment:
I think we could get the replicas number in the function updateStatusNew, because we do not use it here.